Helper/AutomatedTesting.php

```diff
@@ -165,6 +214,53 @@ public function __construct(
         $this->stockRegistry = $stockRegistry;
     }
 
+    /**
+     * Retrieve an Order from the repository
+     *
+     * @return Order|null
+     */
+    protected function getPastOrder()
+    {
+        try {
+            $sortOrder = $this->sortOrderBuilder
+                ->setField('entity_id')
+                ->setDirection('DESC')
+                ->create();
+
+            $nonZeroDiscountTaxSearchCriteria = $this->searchCriteriaBuilder
+                ->addFilter('discount_amount', 0, 'lt')
+                ->addFilter('tax_amount', 0, 'gt')
+                ->setPageSize(1)
+                ->setCurrentPage(1)
+                ->setSortOrders([$sortOrder])
+                ->create();
+
+            $defaultSearchCritieria = $this->searchCriteriaBuilder
+                ->setPageSize(1)
+                ->setCurrentPage(1)
+                ->setSortOrders([$sortOrder])
+                ->create();
+
+            $ordersFound = $this->orderRepository
+                ->getList($nonZeroDiscountTaxSearchCriteria)
```

Comment:
If we are going to select order by other conditions as well maybe we need to consider our own loop by orders instead of `searchCriteriaBuilder`. I am afraid this can be too slow.

Reply:
Actually, we can go ahead for now and optimize performance later.